.github/actions/create-release-candidate-v1/README.md

```diff
@@ -12,6 +12,7 @@ A GitHub Action to create a release candidate.
 | `release-script-path` | Yes      | The path to the [release script](#release-script-requirements) that creates the changelogs and bumps the version of the package(s)                                                                                              | NA      |
 | `version-locked`      | No       | Whether or not the version bump should treat major/minor as "locked". Repos which version-lock to axe-core should default this to `true`, overriding it to `false` only for releases that update `axe-core`.                    | `false` |
 | `docs-repo`           | No       | The name of the repo where the release notes live                                                                                                                                                                               | `null`  |
+| `docs-labels`         | No       | Labels for release notes issue. Comma-delimited list of labels (e.x. "release,PRIORITY: high")                                                                                                                                  | NA      |
```

Comment:
When would the labels ever change for an RC issue that's created? I saw in the ticket it's need for Jazzband but that's happening already, example: https://github.com/dequelabs/docs-developer-hub/issues/116. If we want the labels to be repo specific we should add some defaults as I suspect most will not change from having the `release` and `PRIORITY: HIGH` labels?

Reply:
as discussed with @Zidious  this is not needed